### PR TITLE
Simplifying Datasets.

### PR DESCRIPTION
**Pull request by andrevitorelli**
Closes Issue #46

### CODE REVIEW

**Comment by andrevitorelli** [comment]
The unit tests have to be modified as the data generator they used won't be available anymore.

**Comment by andrevitorelli** [comment]
It's passing! @EiffL .

This PR simplifies the galgen module, adds a module that creates stamps just like the ngmix test, and removes everything that we weren't using of kimages...

File: tests/test_tf_ngmix.py [hunk 1]
```diff
@@ -1,43 +1,127 @@
...
+
 
   results_ngmix = np.array(results_ngmix)
-  
+  gals = tf.stack(imlist)
   # our version:
   @tf.function
   def get_ellipticity(im):
-    return autometacal.get_moment_ellipticities(im, scale=0.2, fwhm=weight_fwhm)
+    return autometacal.get_moment_ellipticities(im, scale=0.263, fwhm=1.2)
```
Comment (EiffL) [inline tests/test_tf_ngmix.py]: why 0.263? note that the header of the file still says `scale = .2`

File: tests/test_tf_ngmix.py [hunk 2]
```diff
@@ -1,43 +1,127 @@
...
+    The shear in each component
+
+  Returns
+  -------
+  ngmix.Observation
+  """
+
+  psf_noise = 1.0e-6
+
+  scale = 0.263
```
Comment (EiffL) [inline tests/test_tf_ngmix.py:34]: inconsistent with what's one line 9

**Review by EiffL — CHANGES_REQUESTED** [review]
looks good but noted some tiny things easy to fix

File: tests/test_tf_ngmix.py (hunk 1 above)
Comment (andrevitorelli) [reply tests/test_tf_ngmix.py]: (same as below)

File: tests/test_tf_ngmix.py (hunk 2 above)
Comment (andrevitorelli) [reply tests/test_tf_ngmix.py:34]: Yes, this seems to be an introduced mistake ~by some github mishandling~.  I'll fix it promptly.

Comment (andrevitorelli) [reply tests/test_tf_ngmix.py:34]: Fixed.

**Comment by andrevitorelli** [comment]
Closed for reorganization of development.

**Pull request closed without merging**